pkgs/os-specific/linux/lvm2/default.nix

```diff
@@ -68,21 +68,18 @@ stdenv.mkDerivation rec {
   '';
 
 
-  patches = lib.optionals stdenv.hostPlatform.isMusl [
+  patches = let
+    version = "eae8e06eee6e93e59139cf4d06e8414f681e20c9";
+  in lib.optionals stdenv.hostPlatform.isMusl [
     (fetchpatch {
       name = "fix-stdio-usage.patch";
-      url = "https://git.alpinelinux.org/aports/plain/main/lvm2/fix-stdio-usage.patch?h=3.7-stable&id=31bd4a8c2dc00ae79a821f6fe0ad2f23e1534f50";
-      sha256 = "0m6wr6qrvxqi2d2h054cnv974jq1v65lqxy05g1znz946ga73k3p";
+      url = "https://git.alpinelinux.org/aports/plain/main/lvm2/fix-stdio-usage.patch?id=${version}";
+      sha256 = "sha256-YNKMmyrbK69N9uGMP+sJ4CMhOqCxthZ/Fg8O+1jMSWI=";
```

Comment:
This requires special attention when updating.

Reply:
Now done unconditionally.